src/tools/rbd_nbd/rbd-nbd.cc

```diff
@@ -98,7 +98,11 @@ static void handle_signal(int signum)
 {
   assert(signum == SIGINT || signum == SIGTERM);
   derr << "*** Got signal " << sig_str(signum) << " ***" << dendl;
-  ioctl(nbd, NBD_DISCONNECT);
+  dout(20) << __func__ << ": " << "sending NBD_DISCONNECT" << dendl;
+  if (ioctl(nbd, NBD_DISCONNECT) < 0)
+    cerr << "rbd-nbd: the device is not used" << std::endl;  
+  else
+    dout(20) << __func__ << ": " << "succesfully disconnect NBD with rbd-nbd" << dendl;
```

Comment:
Please break the line to fit 80 char width. Or rather make the message shorter (e.g. "disconnected" is enough for a debug message).